packages/textlint-formatter/src/formatters/pretty-error.ts

```diff
@@ -0,0 +1,183 @@
+// LICENSE : MIT
+
+// Original code is https://github.com/azer/prettify-error
+// Author : azer
+"use strict";
+import { TextlintMessage, TextlintResult } from "@textlint/kernel";
+import { TextLintFormatterOption } from "../../../textlint/src/textlint-interface";
```

Comment:
It is bad!

Maybe, this is failure reason on CI.

Reply:
@0x6b 
We can just define `TextLintFormatterOption` interface in formatter module.
Because, `textlint` module export `TextLintFormatterOption` interface now.

```ts
interface TextLintFormatterOption {
    formatterName: string;
    color?: boolean;
}
```

📝 Note:

`textlint-formatter` should not depened on `textlint` module.
Because, `textlint` alreadly depended on `textlint-formatter`.
It make a circular reference.

Reply:
@azu, thank you very much. I also found that `textlint-fixer-formatter` defines similar interface on its own.